test/credentials.yml

```diff
@@ -11,9 +11,11 @@ usps:
   login: <%= ENV['ACTIVESHIPPING_USPS_LOGIN'] %>
 
 ups:
-  key: <%= ENV['ACTIVESHIPPING_UPS_LOGIN'] %>
-  login: <%= ENV['ACTIVESHIPPING_UPS_KEY'] %>
-  password: <%= ENV['ACTIVESHIPPING_UPS_PASSWORD'] %>
+  key: 5CD7DE7593CBF516
+  login: apicasperuser
+  password: T30roFStkUyn
+  origin_account: W5A760
+  origin_name: Casper
```

Comment:
Is this intentional? If so, can we use these to run the remote tests in CI? In this case, we should add them to `.travis.yml` instead as environment variables.

I'd prefer to create a separate credentials entry called `ups_origin`, so we can see what tests/functionality requires an origin account, and which one are not.